compose/.apps/plex/plex.armv7l.yml

```
@@ -1,3 +1,3 @@
 services:
   plex:
-    image: ghcr.io/linuxserver/plex:bionic
+    image: ghcr.io/linuxserver/plex:arm32v7-latest
```

```suggestion
    image: ghcr.io/linuxserver/plex
```
The tags just changed recently on the image. We should use the manifest on the `:latest` tag.